ampd/src/config.rs

```diff
@@ -9,15 +9,34 @@ use crate::handlers::{self, config::deserialize_handler_configs};
 use crate::tofnd::Config as TofndConfig;
 use crate::url::Url;
 
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
+pub struct EventProcessorConfig {
+    #[serde(with = "humantime_serde")]
+    pub retry_timeout: Duration,
```

Comment:
```suggestion
    pub retry_delay: Duration,
```
compare to `stream_timeout`, where it fails if the action takes longer than the set duration. Here we wait for x amount of time before retrying

Reply:
Makes sense!